py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,424 @@
+    format_time_left: The format for used for the {time_to} placeholder when when the time
+        displayed is the time left until an event in progress is over.
+        (default '({days}d {hours}h {minutes}m) left')
+    format_time_to: The format used for the {time_to} placeholder.
+        (default '({days}d {hours}h {minutes}m)')
+    num_events: The maximum number of events to display.
+        (default 3)
+    thresholds: Thresholds for event colors. The first entry is the color for event 1,
+        the second for event 2, and so on.
+        (default [])
+    time_to_max: Threshold (in minutes) for when to display the {time_to}
+        string; e.g. if time_to_max = 60, {time_to} will only be displayed
+        for events starting in 60 minutes or less.
+        (default 180)
+    warn_threshold: The number of minutes until an event starts before a warning is
+        displayed to notify the user; e.g. if warn_threshold = 30 and an event is
+        starting in 30 minutes or less, a notification will be displayed. disabled (= 0)
+        by default.
+        (default 0)
+    warn_timeout: The number of seconds before a warning should be displayed again.
+        (default 300)
+
+format placeholders:
+    {events} All the events to display.
+
+format_error placeholders:
+    {error} Error to display.
+
+format_event & format_event_full placeholders:
+    {description} The description for the calendar event.
+    {end_date} The end date for the event.
+    {end_time} The end time for the event.
+    {location} The location for the event.
+    {start_date} The start date for the event.
+    {start_time} The start time for the event.
+    {summary} The summary (i.e. title) for the event.
+    {time_to} The time until the event starts (or until it is over
+        if already in progress).
+
+format_time_to placeholders:
+    {days} The number of days until the event.
+    {hours} The number of hours until the event.
+    {minutes} The number of minutes until the event.
+
+Requires:
+    1. Python library google-api-python-client (can be obtained with pip).
+    2. OAuth 2.0 credentials for the Google Calendar api.
+
+    Follow Step 1 of the guide here to obtain your OAuth 2.0 credentials:
+    https://developers.google.com/google-apps/calendar/quickstart/python
+
+    Download the client_secret.json file which contains your client ID and
+    client secret. In your i3status config file, set configuration parameter
+    client_secret to the path to your client_secret.json file.
+
+    The first time you run the module, a browser window will open asking you
+    to authorize access to your calendar. After authorization is complete,
+    an access/refresh token will be saved to the path configured in
+    auth_token, and i3status will be restarted. This restart will
+    occur only once after the first time you successfully authorize.
+
+Example:
+```
+google_calendar {
+    client_secret = '/path/to/client_secret/'
```

Comment:
we expect files here but we give folders as examples, please adjust this

Reply:
fixed